source/features/parse-backticks.tsx

```diff
@@ -15,8 +15,11 @@ function initGeneral(): void {
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span' // Pinned Issues
-	])) {
+		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'blame-commit-message a', // `isBlame`
+		'.issues_labeled .text-gray-dark > a',
+		'.commits blockquote'
```

Comment:
`//` missing

Reply:
I know I was waiting for you comment above